railstats.py

```
@@ -54,6 +71,12 @@
             counts['arrival_status_by_operator'][row['operator']['code']][row['act_arrival_status']] += 1
         else:
             counts['arrival_status_by_operator'][row['operator']['code']][row['act_arrival_status']] = 1
+        if row['identity'] not in counts['arrival_status_by_identity'].keys():
```

Ruff

Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)